crates/accelerate/src/nlayout.rs

```diff
@@ -24,7 +24,7 @@ use hashbrown::HashMap;
 macro_rules! qubit_newtype {
     ($id: ident) => {
         #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-        pub struct $id(u32);
+        pub struct $id(pub u32);
```

Comment:
This is to unlock what's done in line 1090, right? `PhysicalQubit::new(dag.get_qargs(inst.qubits)[0].0)` (not actionable, I just want to understand it because I will need to access info similarly).

Reply:
Yeah it's so I can access the inner u32 with `.0`.